From 3141133222a728d0735df5d9f8c6ebcd33bfade1 Mon Sep 17 00:00:00 2001
From: Matej Humpal <humpal@medio.cz>
Date: Sun, 9 Jun 2013 20:33:46 +0200
Subject: [PATCH] Ignore all dot files when deleting from temp

---
 mpdf.php | 13 ++++++++-----
 1 file changed, 8 insertions(+), 5 deletions(-)

diff --git a/mpdf.php b/mpdf.php
index b38e00d..d8ea4b3 100644
--- a/mpdf.php
+++ b/mpdf.php
@@ -7201,12 +7201,15 @@ function Output($name='',$dest='')
 	// Delete any files in tmp/ directory that are >1 hrs old
 		$interval = 3600;
 		if ($handle = opendir(preg_replace('/\/$/','',_MPDF_TEMP_PATH))) {
-		   while (false !== ($file = readdir($handle))) { 
-			if (!is_dir($file) && ((filemtime(_MPDF_TEMP_PATH.$file)+$interval) < time()) && ($file != "..") && ($file != ".")) { // mPDF 5.4.19
-				unlink(_MPDF_TEMP_PATH.$file); 
+			while (false !== ($file = readdir($handle))) {
+				if (!is_dir($file)
+						&& ((filemtime(_MPDF_TEMP_PATH.$file)+$interval) < time())
+						&& (0 !== strpos($file, '.')
+				){ // mPDF 5.4.19
+					unlink(_MPDF_TEMP_PATH.$file);
+				}
 			}
-		   }
-		   closedir($handle); 
+			closedir($handle);
 		}
 	//==============================================================================================================
 
-- 
1.8.3

